### ISSUE TITLE
Speed limit is shown in mph even when switching to km/h

### ISSUE THREAD

**Issue by cgadam** [plusCubed/velociraptor#59]
I configured Velociraptor to show speed in km/h and even though the speed itself is expressed in km/h the speed limit is still expressed in mph.

**Comment by plusCubed** [comment]
Does this still occur on the latest version 1.5.4?

**Comment by TommyB28** [comment]
Yes I have the same problem. I am in Australia. The speed is working fine but the SPEED LIMIT is being converted to MPH. e.g. in a 50kph zone the app shows 31 as the limit and in a 60kph zone, the app shows 37. Apart from that, it's awesome.

**Comment by CrixuAMG** [comment]
I also have this issue, latest version from the play store.
